src/styles/index.scss

```
@@ -1,3 +1,66 @@
+$padding: 10px;
+$cellSize: 100px;
+$gap: 1px;
+$dayNumber: 7;
+
+@function blockwidth($padding, $dayWidth, $gap, $columnNumber) {
```

```suggestion
@function getBlockwidth($padding, $dayWidth, $gap, $columnNumber) {
```
Function name should start from verb